### PR TITLE
[FlowBundle] missing specs and CS

### PR DESCRIPTION
**Pull request by aRn0D**
| Q             | A
| ------------- | ---
| Bug fix?      | no
| New feature?  | refactoring 
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | no
| License       | MIT
| Doc PR        | -

### CODE REVIEW

File: src/Sylius/Bundle/InstallerBundle/Process/InstallerScenario.php [hunk 1]
```diff
@@ -23,10 +23,10 @@ class InstallerScenario extends ContainerAware implements ProcessScenarioInterfa
...
     {
         $builder
-            ->add('welcome', new Step\WelcomeStep())
-            ->add('check', new Step\CheckStep())
-            ->add('configure', new Step\ConfigureStep())
-            ->add('setup', new Step\SetupStep())
+            ->add('welcome', new Step\WelcomeStepAbstract())
+            ->add('check', new Step\CheckStepAbstract())
+            ->add('configure', new Step\ConfigureStepAbstract())
+            ->add('setup', new Step\SetupStepAbstract())
```
Comment (michalmarcinkowski) [inline src/Sylius/Bundle/InstallerBundle/Process/InstallerScenario.php]: I think that change was unwanted :)

Comment (aRn0D) [inline src/Sylius/Bundle/InstallerBundle/Process/InstallerScenario.php]: yes I will revert it !

**Comment by michalmarcinkowski** [comment]
@aRn0D and one more thing... We should avoid fluent interfaces unless there is a really good reason for that. Related article http://ocramius.github.io/blog/fluent-interfaces-are-evil/, hope it will convince you.

You're doing really good job! :+1:

**Comment by pjedrzejewski** [comment]
Agree with @michalmarcinkowski about fluent interfaces. :)

**Comment by aRn0D** [comment]
@michalmarcinkowski @pjedrzejewski updated !

**Comment by aRn0D** [comment]
ping @pjedrzejewski :p

**Comment by michalmarcinkowski** [comment]
:+1:

**Comment by aRn0D** [comment]
ping @pjedrzejewski

**Comment by pjedrzejewski** [comment]
I'm not sure it was a good idea to rewrite these tests in specs? Specs have biggest benefit when they drive the design and code, otherwise they are very similar to phpunit tests, except with a nicer syntax and easier mocking/modification in future. That being said, you did it already so it is good! Thank you Arnaud!

**Comment by aRn0D** [comment]
It is for consistency and for future development!

**Comment by umpirsky** [comment]
This PR is a BC break since it renames `ControllerStep`.

Added to https://github.com/Sylius/Sylius/pull/3393.

**Pull request merged**